cmd/containerboot/main_test.go

```diff
@@ -346,6 +346,30 @@ func TestContainerBoot(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name: "kube_disk_storage",
+			Env: map[string]string{
+				"KUBERNETES_SERVICE_HOST":       kube.Host,
+				"KUBERNETES_SERVICE_PORT_HTTPS": kube.Port,
+				"TS_KUBE_SECRET":                "",
```

Comment:
add comment like `// testing that it's explicitly set to an empty value` ?

Reply:
Done.